src/Afsy/Bundle/FrontBundle/Controller/BlogController.php

```diff
@@ -26,7 +26,7 @@ public function indexAction()
         $query = $this->getDoctrine()->getRepository('AfsyCoreBundle:Article')->getQuery();
         $pagination = $this->get('knp_paginator')->paginate(
             $query,
-            $this->get('request')->query->get('page', 1)
+            $this->get('request_stack')->getCurrentRequest()->query->get('page', 1)
```

Comment:
you can pass the request object as a controller argument. It seems better because it is managed by the framework bundle.